diff --git a/node_modules/@steambrew/client/build/index.d.ts b/node_modules/@steambrew/client/build/index.d.ts
index 28bd18f..1ddfccb 100644
--- a/node_modules/@steambrew/client/build/index.d.ts
+++ b/node_modules/@steambrew/client/build/index.d.ts
@@ -23,5 +23,5 @@ export interface Plugin {
     alwaysRender?: boolean;
     titleView?: JSX.Element;
 }
-export type DefinePluginFn = () => Plugin;
+export type DefinePluginFn = () => Promise<Plugin>;
 export declare const definePlugin: (fn: DefinePluginFn) => DefinePluginFn;
