diff --git a/node_modules/@steambrew/ttc/dist/index.js b/node_modules/@steambrew/ttc/dist/index.js
index b3c22b7..f5ed3a0 100755
--- a/node_modules/@steambrew/ttc/dist/index.js
+++ b/node_modules/@steambrew/ttc/dist/index.js
@@ -200,7 +200,7 @@ function ExecutePluginModule() {
     }
     /** Expose the OnPluginConfigChange so it can be called externally */
     MillenniumStore.OnPluginConfigChange = OnPluginConfigChange;
-    MILLENNIUM_BACKEND_IPC.postMessage(0, { pluginName: pluginName, methodName: '__builtins__.__millennium_plugin_settings_parser__' }).then((response) => {
+    MILLENNIUM_BACKEND_IPC.postMessage(0, { pluginName: pluginName, methodName: '__builtins__.__millennium_plugin_settings_parser__' }).then(async (response) => {
         /**
          * __millennium_plugin_settings_parser__ will return false if the plugin has no settings.
          * If the plugin has settings, it will return a base64 encoded string.
@@ -220,7 +220,7 @@ function ExecutePluginModule() {
             __millennium_internal_plugin_name_do_not_use_or_change__: pluginName,
         });
         /** Run the rolled up plugins default exported function */
-        let pluginProps = PluginModule.default();
+        let pluginProps = await PluginModule.default();
         function isValidSidebarNavComponent(obj) {
             return obj && obj.title !== undefined && obj.icon !== undefined && obj.content !== undefined;
         }
